documentation/rfc/ridesharing.md

```diff
@@ -0,0 +1,115 @@
+# Ridesharing
+
+
+## 1-1 call over a simple crowfly
+
+The simplest way of dealing with ridesharing is to make a crowfly,
+then call an external system to "cover" that crowfly section.
+
+
+### Output description
+
+The output of a `/journeys` call would follow [this json example](./journeys_ridesharing.json).
+
+Simplified output:
+```json
+{
+    "journeys": [
+        {
+            "requested_date_time": "20180101T070000",
+            "sections": [
+                {
+                    "type": "crow_fly",
+                    "mode": "ridesharing",
+                    "from": "A",
+                    "to": "B",
+                    "departure_date_time": "20180101T070000",
+                    "arrival_date_time": "20180101T090000",
+                    "ridesharing_journeys": [
+                        {
+                            "sections":[
+                                {
+                                    "from": "A",
+                                    "to": "A1",
+                                    "departure_date_time": "20180101T063000",
+                                    "arrival_date_time": "20180101T063000",
+                                    "type": "crow_fly",
+                                    "mode": "walking"
+                                },
+                                {
+                                    "from": "A1",
+                                    "to": "A2",
+                                    "departure_date_time": "20180101T063000",
+                                    "arrival_date_time": "20180101T093000",
+                                    "type": "ridesharing"
```

Comment:
We have a mode and a type named "ridesharing", it's strange, this can probably be a "street_network" section.

Reply:
We won't have streetnetwork's usual informations (unless we recompute the path) so I'm not in on that one.